### ISSUE TITLE
Meaning of num_groups

### ISSUE THREAD

**Issue by claell** [ranahanocka/MeshCNN#65]
I don't know what the `num_groups` parameter does. At first I thought it is the number of classes used for training, but apparently that is not correct.

**Comment by ranahanocka** [comment]
It is a hyper-parameter for the groupnorm normalization layer. You can see more info in the [PyTorch docs for GroupNorm](https://pytorch.org/docs/stable/nn.html#groupnorm).

**Comment by claell** [comment]
Thanks for the explanation. I should have looked into the options file [here](https://github.com/ranahanocka/MeshCNN/blob/master/options/base_options.py#L26).
